source/MaterialXCore/Document.cpp

```diff
@@ -1422,6 +1422,16 @@ void Document::upgradeVersion()
             }
         }
 
+        for (LookGroupPtr lookGroup : getLookGroups())
+        {
+            if (lookGroup->hasAttribute("active"))
```

Comment:
Nitpicks: Declare a constant string for "active"  outside the look, and `string active` I assume should be const.